.travis.yml

```diff
@@ -3,7 +3,7 @@ os:
     - osx
     - linux
 julia:
-    - 0.5
+    - 0.6
     - nightly
```

Comment:
test on 0.7 as well? (nightly is going to diverge soon)

Reply:
Sure, if 0.7 binaries are available on Travis already. Are they?

Reply:
Yes